scripts/publish.sh

```diff
@@ -5,6 +5,8 @@ source $OKTA_HOME/$REPO/scripts/setup.sh
 REGISTRY="${ARTIFACTORY_URL}/api/npm/npm-okta"
 
 export TEST_SUITE_TYPE="build"
+yarn global add @okta/ci-update-package
```

Comment:
hmm, how does it work before when publishing 2.x?

Reply:
I am not sure how it worked before. With this change I am trying to solve an issue we are having the publish script not being able to run with version 1.17.0.  @manueltanzi-okta pointed me to the latest publish script that has these change https://github.com/okta/okta-auth-js/blob/master/scripts/publish.sh#L11

Reply:
I see. because you're fixing on the release branch.